website/addons/s3/static/s3UserConfig.js

```diff
@@ -30,13 +30,52 @@ function ViewModel(url) {
     self.accounts = ko.observableArray();
 
     ChangeMessageMixin.call(self);
+    self.options = {
```

Comment:
So you got rid of s3NodeConfig and replaced it with node-cfg but you're keeping s3UserConfig?